torch/testing/_internal/common_methods_invocations.py

```diff
@@ -2700,6 +2700,37 @@ def sample_inputs_fliplr_flipud(op_info, device, dtype, requires_grad, **kwargs)
     )
     return [SampleInput(tensor) for tensor in tensors]
 
+def sample_inputs_fmod_remainder(op_info, device, dtype, requires_grad, **kwargs):
+    make_arg = partial(make_tensor, dtype=dtype, device=device, requires_grad=requires_grad)
+
+    cases = (
+        ((S, S, S), (), 1.5, False),
+        ((), 1.5, (), False),
+        ((S, S, S), (S, S, S), 1.5, False)
+    )
+
+    # Sample inputs with broadcasting
+    cases_with_broadcasting = (
+        ((S,), (S, S, S), 1.5, True),
+        ((S, S, S), (S,), 1.5, True),
```

Comment:
I think this sample is wrong. As `rhs` will be broadcasted but not the `lhs`.

Reply:
Thanks for pointing out. So these samples should just be removed? Removing them from `broadcasting` will raise errors like:

```
RuntimeError: output with shape [5, 1, 5] doesn't match the broadcast shape [5, 5, 5]
```

Reply:
Move them to the above `cases`.

Eg.

```python
cases = (
        ((S, S, S), (), 1.5, False),
        ((), 1.5, (), False),
        ((S, S, S), (S, S, S), 1.5, False),
        ((S, S, S), 1.5, (), False), # Added from below
        ((S, S, S), (S,), 1.5, False), # Added from below
    )
```

Reply:
Done, thanks!